share/scripts/inverse/update_imc.sh

```diff
@@ -38,7 +37,7 @@ for group in $imc_groups; do
   # currently this is a hack! need to create combined array
   msg "solving linear equations for imc group '$group'"
   critical csg_imcrepack --in ${group} --out ${group}.packed
-  do_external imcsolver $solver  ${group}.packed ${group}.packed.sol $reg
+  critical csg_imc_solve --imcfile "${group}.packed.imc" --gmcfile "${group}.packed.gmc" --regularization "${reg}" --outputfile "${group}.packed.sol"
   critical csg_imcrepack --in ${group}.packed --unpack ${group}.packed.sol
```

Comment:
@JensWehner, can be combine this series of `csg_imcrepack`, `csg_imc_solve`, `csg_imcrepack` into one thing?

Reply:
I put the input files to `csg_imcrepack` here: https://github.com/votca/csg/issues/407#issuecomment-533553172

Reply:
can you add the final output as well?

Reply:
Without the `--unpack` option, it generates the `CG-CG.packed` files, with the `--unpack` option is generates the potentials e.g. *.imc.dpot`.

I uploaded a complete set here:
https://github.com/votca/csg/issues/407#issuecomment-533628560